src/ipc/ipc4/handler.c

```diff
@@ -1040,7 +1040,7 @@ void ipc_cmd(struct ipc_cmd_hdr *_hdr)
 		return;
 
 	if (cpu_is_primary(cpu_get_id()))
-		tr_info(&ipc_tr, "rx\t: %#x|%#x", in->primary.dat, in->extension.dat);
+		tr_dbg(&ipc_tr, "rx\t: %#x|%#x", in->primary.dat, in->extension.dat);
```

Comment:
NAK, please, keep this

Reply:
I know this is debug ipc key message, I think this part code already matured, and no need further debug, this is the reason for remove this print.

Reply:
no, it isn't because we are unsure whether this print is working, it is because this is the primary means of matching firmware and kernel logs, please, keep.